test/test_benchmark.jl

```diff
@@ -191,3 +191,14 @@ end
 
     @test truth ≈ s
 end
+
+@testset "Test Manifest.toml being used" begin
+    # Ensure AirspeedVelocity-ManifestTest.jl has its Manifest.toml copied:
+    # Test with CLI version:
+    @test nothing === benchpkg(
+        "AirspeedVelocity_ManifestTest";
+        rev="main",
+        url="https://github.com/Zentrik/AirspeedVelocity_ManifestTest",
+        use_manifest=true
+    )
+end
```

Comment:
Could this test be added directly within AirspeedVelocity.jl? Maybe just a package within the `test` folder? I'm a bit worried about the tests living in another repo

Reply:
Immediately after saying this I realized that all the other tests also have this issue... hmm... Maybe it requires a refactor in the future.

But maybe this test is simple enough we could make a special module for it?

Reply:
I think I can just copy the test package into a subdirectory in this repo and fetch it from here, let me see.